.github/workflows/compliance.yml

```diff
@@ -69,8 +69,8 @@ jobs:
         # debug
         ls -la
         git log  --pretty=oneline | head -n 10
-        ./scripts/ci/check_compliance.py --annotate -m Devicetree -m Gitlint \
-        -m Identity -m Nits -m pylint -m checkpatch -m Kconfig \
+        ./scripts/ci/check_compliance.py --annotate -m DevicetreeBindings
```

Comment:
dont you need a \ here?

Reply:
I do, thanks!

Reply:
I think you fixed this in a later commit

Reply:
Hmm wait, this was fixed after in fact.